config/webhook/manifests.yaml

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
This file is generated from `//+kubebuilder:webhook` marker. The rest of files in `config/webhook` were copied from a kubebuilder-style webhook I generated on a side.